lib/findExports.js

```diff
@@ -37,6 +37,31 @@ function findESNamedExports(node) {
   return result;
 }
 
+function resolveNestedNamedExports(node, absolutePathToFile) {
+  if (node.type === 'ConditionalExpression') {
+    // Potential ternary-style export - we pick the first one
+    // module.exports = foo ? require('a') : require('b');
+    return resolveNestedNamedExports(node.consequent, absolutePathToFile);
+  }
+  if (
+    node.type === 'CallExpression' &&
+    node.callee.name === 'require' &&
+    node.arguments.length === 1 &&
+    node.arguments[0].type === 'StringLiteral'
```

Comment:
Yeah, there are definitely some room for improvements here. I find it a little hard to name though in some cases. The names in the AST are pretty abstract already, and in some cases we look for even more abstract constructs. I'll remember this for next time in here though.